src/Expression/ArrayExpressionBuilder.php

```diff
@@ -100,34 +108,34 @@ protected function buildPlaceholders(ExpressionInterface $expression, &$params):
 
     private function unnestArrayExpression(ArrayExpression $expression, $value): ArrayExpression
     {
-        $expressionClass = \get_class($expression);
+        $expressionClass = get_class($expression);
 
         return new $expressionClass($value, $expression->getType(), $expression->getDimension() - 1);
     }
 
-    protected function getTypehint(ArrayExpression $expression): string
+    protected function getTypeHint(ArrayExpression $expression): string
     {
         if ($expression->getType() === null) {
             return '';
         }
 
         $result = '::' . $expression->getType();
-        $result .= \str_repeat('[]', $expression->getDimension());
+        $result .= str_repeat('[]', $expression->getDimension());
 
         return $result;
     }
 
     /**
-     * Build an array expression from a subquery SQL.
+     * Build an array expression from a subQuery SQL.
```

Comment:
"subquery" is a single word. See the following SQL'99 standard chapter: https://crate.io/docs/sql-99/en/latest/chapters/31.html

Reply:
done.